CMake/Dependencies.cmake

```diff
@@ -42,11 +59,21 @@ else()
 endif()
 
 add_library(DevilutionX::SDL INTERFACE IMPORTED GLOBAL)
-if(USE_SDL1)
-  target_link_libraries(DevilutionX::SDL INTERFACE ${SDL_LIBRARY})
-  target_compile_definitions(DevilutionX::SDL INTERFACE USE_SDL1)
+if(UWP_LIB)

```

Comment:
Perhaps this can be simplified by simply not setting or unsetting `SDL2_MAIN` in case of `UWP_LIB`?